client/src/routes/login/tabs/SignUp.tsx

```diff
@@ -40,41 +46,88 @@ function SignUp () {
 	
 	const [passwordsMatch, setPasswordsMatch] = React.useState(true);
 
+	const [errorText, setErrorText] = React.useState('');
+
+	const getLeeftijdsGroep = (birthYear:any) => {
+		const estimatedAge = new Date().getFullYear() - birthYear;
+
+		if(estimatedAge < 18) {
```

Comment:
I think you can make an enum for the agegroups, or maybe we already have one double check that